client-event-handling/sample-vsce/package.json

```diff
@@ -0,0 +1,165 @@
+      {
+        "command": "menu-item-sample.unsubscribeEU",
+        "title": "Unsubscribe[EU]"
+      },
+      {
+        "command": "menu-item-sample.emitZU",
+        "title": "Emit[ZU]"
+      },
+      {
+        "command": "menu-item-sample.subscribeZU",
+        "title": "Subscribe[ZU]"
+      },
+      {
+        "command": "menu-item-sample.unsubscribeZU",
+        "title": "Unsubscribe[ZU]"
+      },
+      {
+        "command": "menu-item-sample.emitZS",
+        "title": "Emit[ZS]"
+      },
+      {
+        "command": "menu-item-sample.subscribeZS",
+        "title": "Subscribe[ZS]"
+      },
+      {
+        "command": "menu-item-sample.unsubscribeZS",
+        "title": "Unsubscribe[ZS]"
+      }
+    ],
+    "menus": {
+      "view/item/context": [
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.subscribeES",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.emitES",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.unsubscribeES",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.subscribeEU",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.emitEU",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.unsubscribeEU",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.subscribeZS",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.emitZS",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.unsubscribeZS",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.subscribeZU",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.emitZU",
+          "group": "100_zowe_menuItemSample@0"
+        },
+        {
+          "when": "view =~ /^zowe\\./",
+          "command": "menu-item-sample.unsubscribeZU",
+          "group": "100_zowe_menuItemSample@0"
+        }
+      ]
+    }
+  },
+  "extensionDependencies": [
+    "Zowe.vscode-extension-for-zowe"
+  ],
+  "resolutions": {
+    "@zowe/imperative": "https://gitpkg.now.sh/zowe/zowe-cli/packages/imperative?ac67acc9d01ec2514c1cb3a022c13a1c5d85a79e"
```

Comment:
Thanks for adding instructions to the readme for how to build with a commit hash of Imperative 😍 

Now that the Event Emitter PR has been merged, I would think that most folks who check out the sample don't need to care about the resolution anymore. Would it make sense to remove it from package.json but leave the note about it in the readme?

Reply:
Makes sense, I think the resolution (and the note) can be removed since we already have a working/published version out on npm that people can use here 😋